### ISSUE TITLE
build uses old config if killed with ctrl-c and restarted

### ISSUE THREAD

**Issue by aeyakovenko** [kerl/kerl#158]
I killed kerl to install some dependencies.  After installing them i still saw this message:

```
$ KERL_CONFIGURE_DISABLE_APPLICATIONS=odbc KERL_CONFIGURE_OPTIONS="--enable-dirty-schedulers --disable-hipe --with-openssl" kerl build 19.0 erlang.19.0.dirty_sched.no_hipe.ssl
Verifying archive checksum...
Checksum verified (f64eefc697d47193444aa224d971f367)
Building Erlang/OTP 19.0 (erlang.19.0.dirty_sched.no_hipe.ssl), please wait...
APPLICATIONS DISABLED (See: /home/ayakovenko/.kerl/builds/erlang.19.0.dirty_sched.no_hipe.ssl/otp_build_19.0.log)
 * crypto         : No usable OpenSSL found
 * jinterface     : No Java compiler found
 * odbc           : ODBC library - link check failed
 * ssh            : No usable OpenSSL found
 * ssl            : No usable OpenSSL found
APPLICATIONS INFORMATION (See: /home/ayakovenko/.kerl/builds/erlang.19.0.dirty_sched.no_hipe.ssl/otp_build_19.0.log)
 * wx             : wxWidgets not found, wx will NOT be usable
DOCUMENTATION INFORMATION (See: /home/ayakovenko/.kerl/builds/erlang.19.0.dirty_sched.no_hipe.ssl/otp_build_19.0.log)
 * documentation  : 
 *                  xsltproc is missing.
 *                  fop is missing.
 *                  The documentation can not be built.
```

after `rm -rf .kerl/builds/erlang.19.0.dirty_sched.no_hipe.ssl` it worked fine

**Comment by mrallen1** [comment]
I see - you let the `configure` script run to completion and then you restarted kerl and it (under the hood) reran `configure` and used its cache'd values for configuration options.

Thanks for the report. I agree that's something we ought to try and improve.

The easiest fix for this would probably be to remove the cache file (if it exists) before `configure` is run.

**Comment by mrallen1** [comment]
This issue should be addressed by #177 so I'm going to close this issue. Please reopen it if there are still problems. Thanks!
